### PR TITLE
Add support for RedisSentinel broker

### PR DESCRIPTION
**Pull request by fjcapdevila**
Add basic support for Redis + Sentinel as broker.

### CODE REVIEW

File: flower/utils/broker.py [hunk 1]
```diff
@@ -203,6 +246,8 @@ def __new__(cls, broker_url, *args, **kwargs):
             return RedisSsl(broker_url, *args, **kwargs)
         elif scheme == 'redis+socket':
             return RedisSocket(broker_url, *args, **kwargs)
+        elif scheme == 'sentinel':
```
Comment (mher) [inline flower/utils/broker.py]: How about using `redis+sentinel` as a protocol name?

Comment (fjcapdevila) [reply flower/utils/broker.py]: I've decided to use that protocol name for consistency with Celery docs:
https://docs.celeryproject.org/en/latest/getting-started/brokers/redis.html#configuration

**Comment by doncraig** [comment]
Is there any documentation available on how to configure master_name in Docker?

**Comment by vuducmanh11** [comment]
> Is there any documentation available on how to configure master_name in Docker?

are you configured it? :(

**Pull request merged**